workers/unity/Packages/io.improbable.gdk.core/Systems/EcsViewSystem.cs

```diff
@@ -16,6 +16,10 @@ public class EcsViewSystem : ComponentSystem
             new Dictionary<uint, IEcsViewManager>();
 
         private WorkerSystem worker;
+        
+        private ProfilerMarker applyDiffMarker = new ProfilerMarker("EcsViewSystem.ApplyDiff");
+        private ProfilerMarker addEntityMarker = new ProfilerMarker("EcsViewSystem.OnAddEntity");
+        private ProfilerMarker removeEntityMarker = new ProfilerMarker("EcsViewSystem.OnRemoveEntity");
```

Comment:
If would be neat if these could be annotations instead given we just wrap the entire method body in the using block. 

Unity pls

Reply:
Its on my TODO list once we have the IL post processor